CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Change Log
 ==========
 
+1.12.0 (2018-09-25)
```

Comment:
Please leave the date as `unreleased` -- this PR isn't creating the release, only adding commits which will be eventually part of it.

Reply:
Roger, that makes sense